gdnative-derive/src/lib.rs

```diff
@@ -164,6 +197,44 @@ pub fn profiled(meta: TokenStream, input: TokenStream) -> TokenStream {
 /// - `no_editor`
 ///
 /// Hides the property from the editor. Does not prevent it from being sent over network or saved in storage.
+///
+/// ### `#[methods]`
+/// Adds the necessary information to an `impl` block for 
+///
+/// ### `#[export]`
+/// Registers the attributed function signature to be used by Godot.
+/// A valid function signature must 
```

Comment:
"...must have:"

(add "have" and colon, otherwise it's not a complete sentence)